services/src/main/java/io/druid/cli/CliMiddleManager.java

```diff
@@ -58,7 +58,7 @@
  */
 @Command(
     name = "middleManager",
-    description = "Runs a Middle Manager, this is a \"task\" node used as part of the remote indexing service."
+    description = "Runs a Middle Manager, this is a \"task\" node used as part of the remote indexing service, see http://druid.io/docs/latest/Middlemanager.html for a description"
```

Comment:
http://druid.io/docs/latest/design/middlemanager.html is a more up-to-date link.

Reply:
done